packages/platforms/delta/arm64/tn48m/src/arm64_delta_tn48m/module/src/platform_lib.c

```diff
@@ -108,7 +109,23 @@ plat_info_t gPlat_info[] = {
 
         .thermal_count  = 11,
         .fan_count = 3,
-        .psu_count = 2,
+        .psu_count = 3,
```

Comment:
Please factor out changing an existing platform into a separate commit.

Reply:
Done.

Reply:
Thank you, but it’d be great if you rephrased the commit message. Maybe something like:

> With the new device model …, a third PSU has to be added, so increase and add the third PSU type.